spec/Appendix B -- Grammar Summary.md

```diff
@@ -194,10 +194,10 @@ TypeSystemExtension :
   - SchemaExtension
   - TypeExtension
 
-SchemaDefinition : schema Directives[Const]? { OperationTypeDefinition+ }
+SchemaDefinition : schema Directives[Const]? { RootOperationTypeDefinition+ }
 
 SchemaExtension :
-  - extend schema Directives[Const]? { OperationTypeDefinition+ }
+  - extend schema Directives[Const]? { RootOperationTypeDefinition+ }
   - extend schema Directives[Const]
 
 OperationTypeDefinition : OperationType : NamedType
```

Comment:
Wouldn't you also need to change this to `RootOperationTypeDefinition`?

Reply:
@mjmahone Good catch, fixed. Thanks for review 👍